src/configs/FirebaseConfig.js

```diff
@@ -35,6 +35,7 @@ const FirebaseConfig = (props) => {
     if (dynamicLink?.url?.includes('postPrivateId=')) return handlePrivatePost(dynamicLink);
     if (dynamicLink?.url?.includes('communityName')) return handleCommunityPage(dynamicLink);
     if (dynamicLink?.url?.includes('users?username=')) return getUserProfile(dynamicLink?.url);
+    if (dynamicLink?.url?.includes('?username=')) return getUserProfileV3(dynamicLink?.url);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for `?username=` in the URL might be too broad and could potentially match URLs that are not intended to be handled by `getUserProfileV3`. It would be better to use a more specific pattern that matches the expected format of the URLs that this function should handle.
```diff
-    if (dynamicLink?.url?.includes('?username=')) return getUserProfileV3(dynamicLink?.url);
+    if (dynamicLink?.url?.startsWith('https://yourapp.com/?username=')) return getUserProfileV3(dynamicLink?.url);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->